time/timer.go

```diff
@@ -0,0 +1,649 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package gxtime encapsulates some golang.time functions
+package gxtime
+
+import (
+	"container/list"
+	"fmt"
+	"log"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+import (
+	uatomic "go.uber.org/atomic"
+)
+
+var (
+	ErrTimeChannelFull   = fmt.Errorf("timer channel full")
+	ErrTimeChannelClosed = fmt.Errorf("timer channel closed")
+)
+
+// Init initializes a default timer wheel
+func Init() {
+	defaultTimerWheelOnce.Do(func() {
+		defaultTimerWheel = NewTimerWheel()
+	})
+}
+
+// Now returns the current time.
+func Now() time.Time {
+	return defaultTimerWheel.Now()
+}
+
+////////////////////////////////////////////////
+// timer node
+////////////////////////////////////////////////
+
+var (
+	defaultTimerWheelOnce sync.Once
+	defaultTimerWheel     *TimerWheel
+	nextID                TimerID
+	curGxTime             = time.Now().UnixNano() // current goext time in nanoseconds
+)
+
+const (
+	maxMS     = 1000
+	maxSecond = 60
+	maxMinute = 60
+	maxHour   = 24
+	maxDay    = 31
+	// ticker interval不能设置到这种精度，
```

Comment:
change to english

Reply:
this var is private local var. what's more, my english is not so good that i can translate it into accurate english text to let user know what I mean. maybe u can help me.

Reply:
done